### ISSUE TITLE
Barrel with 64 blocks in it wont automatically accept items

### ISSUE THREAD

**Issue by Gamertech** [LatvianModder/YABBA#103]
MC- 1.12.2
YABBA-1.1.2.25
FTBlib- 5.3.0.46

When a standard, non-upgraded barrel has exactly 64 items (1 stack) in it, it wont accept any more auto-inserted items from a hopper or conduit until you change it, such as manually adding a single item to put it over the full stack (1x64+1).

It will auto-fill up to 64 before blocking any more items. If it's manually put over 1 stack, it will continue to auto-fill as expected until full.

Has lasted over several mod and forge versions.

**Comment by LatvianModder** [comment]
Yeah, it's a dumb check in hopper, I have an idea how to fix it though
